lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -64,6 +69,7 @@ internal sealed class ContextRuntime : IDisposable
                 Optional<ContextRuntime> parentContext)
         {
             _serviceInjector = serviceInjector;
+            _injectedServices = serviceInjector.GetNamedInstance<ServicesSet, ISet<object>>();
```

Comment:
Does an injector pass itself into the constructor when calling `GetInstance`? I'm not sure if that is how it works. i.e., when we call `serviceInjector.GetInstace<ContextRuntime>`, do we get `serviceInjector` for free in the constructor of `ContextRuntime`?